relay-event-normalization/src/event.rs

```diff
@@ -200,15 +200,17 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -
     transactions_processor.process_event(event, meta, ProcessingState::root())?;
 
     // Check for required and non-empty values
-    schema::SchemaProcessor.process_event(event, meta, ProcessingState::root())?;
+    schema::SchemaProcessor
+        .process_event(event, meta, ProcessingState::root())
+        .ok();
```

Comment:
Personally I really don't like ignoring errors via `.ok()` this is just a combinator that returns an `Option` instead of `Result` which happens to silence the `must_use`. The `let _ = ...` syntax is a lot more obvious imo.

Reply:
I don't think there is a convention in Relay for this, but I agree `.ok()` is not very obvious. Addressed in [e1e008d](https://github.com/getsentry/relay/pull/2935/commits/e1e008dbabbe87140fdfaef9cdda5d34c148ae4b).